src/easydiffraction/sample_models/sample_model.py

```diff
@@ -0,0 +1,210 @@
+from easydiffraction.utils.formatting import paragraph
+from easydiffraction.core.objects import Datablock
+from easydiffraction.crystallography import crystallography as ecr
+from easydiffraction.sample_models.components.space_group import SpaceGroup
+from easydiffraction.sample_models.components.cell import Cell
+from easydiffraction.sample_models.collections.atom_sites import AtomSites
+
+
+class SampleModel(Datablock):
+    """
+    Represents an individual structural model of a sample.
+    Wraps crystallographic information including space group, cell, and
+    atomic sites.
+    """
+
+    # TODO: Move cif_path and cif_str out of __init__ and into separate methods
+    def __init__(self,
+                 name: str,
+                 cif_path: str = None,
+                 cif_str: str = None):
+        super().__init__()
+        self._name = name
+        self.space_group = SpaceGroup()
+        self.cell = Cell()
+        self.atom_sites = AtomSites()
+
+        if cif_path:
+            self.load_from_cif_file(cif_path)
+        elif cif_str:
+            self.load_from_cif_string(cif_str)
+
```

Comment:
no `else`? What is going to happen when neither `cif_path` nor `cif_str` is passed?

Reply:
Good point. I haven’t touched this part yet, since it feels more relevant to handle it properly as part of task #21.

Reply:
Then please add a `TODO` so it doesn't slip from implementing and we forget to re-check this later.

Reply:
I've referenced this conversation in issue #21 to ensure it's addressed as part of that task.